<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client_menu</class>
 <widget class="QMainWindow" name="Client_menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Logo_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>521</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:48pt;&quot;&gt;Logo&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>214</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="Back_Button">
         <property name="styleSheet">
          <string notr="true">background-color : rgb(68, 114, 196);
border-style : outset;
border-width : 5px;
border-radius : 20px;
border-color : rgb(129, 143, 168);
font: 28pt &quot;배달의민족 주아&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Home_Button">
         <property name="styleSheet">
          <string notr="true">background-color : rgb(68, 114, 196);
border-style : outset;
border-width : 5px;
border-radius : 20px;
border-color : rgb(129, 143, 168);
font: 28pt &quot;배달의민족 주아&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="Manager_Button">
       <property name="styleSheet">
        <string notr="true">background-color : rgb(68, 114, 196);
border-style : outset;
border-width : 5px;
border-radius : 10px;
border-color : rgb(129, 143, 168);
font: 20pt &quot;배달의민족 주아&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>관리자 모드</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Courier_Button">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>310</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(68, 114, 196);
border-style : outset;
border-width : 5px;
border-radius : 20px;
border-color : rgb(129, 143, 168);
font: 28pt &quot;배달의민족 주아&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>택배기사</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Client_Button">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>200</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(68, 114, 196);
border-style : outset;
border-width : 5px;
border-radius : 20px;
border-color : rgb(129, 143, 168);
font: 28pt &quot;배달의민족 주아&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>일반고객</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_date">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>520</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;배달의민족 주아&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;label_date&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_time">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>520</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;배달의민족 주아&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;label_time&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Storage_Box_Button">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>420</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(68, 114, 196);
border-style : outset;
border-width : 5px;
border-radius : 20px;
border-color : rgb(129, 143, 168);
font: 28pt &quot;배달의민족 주아&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>보관함 현황</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
